app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>USER NR: <%=@user.id%></h1>
+<% if current_user? %>
```

Comment:
Authorization (restricting what user can see) should be handled on a controller level. I'd suggest using pundit (https://github.com/elabs/pundit).